components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/generic-exception-response.jsp

```diff
@@ -43,12 +42,12 @@
 
     <div id="workArea">
         <div class="info-box">
-            <%=stat%>
+            <%=Encode.forHtmlContent(stat)%>
```

Comment:
Use Encode.forHtml here since we can embed HTML tags within <div> tags. So it's important to encode for all significant HTML characters